src/core/api.ts

```diff
@@ -21,11 +20,19 @@ interface GeneSuggestion {
 export interface GoslingApi {
     subscribe: typeof subscribe;
     unsubscribe: typeof unsubscribe;
-    zoomTo(viewId: string, position: string, padding?: number, duration?: number): void;
-    zoomToExtent(viewId: string, duration?: number): void;
-    zoomToGene(viewId: string, gene: string, padding?: number, duration?: number): void;
-    suggestGene(viewId: string, keyword: string, callback: (suggestions: GeneSuggestion[]) => void): void;
+    zoomTo(trackId: string, position: string, padding?: number, duration?: number): void;
+    zoomToExtent(trackId: string, duration?: number): void;
+    zoomToGene(trackId: string, gene: string, padding?: number, duration?: number): void;
+    suggestGene(trackId: string, keyword: string, callback: (suggestions: GeneSuggestion[]) => void): void;
+    /**
+     * This is for the backward compatibility and will be deprecated. Use `getTrackIds()` instead.
+     */
     getViewIds(): string[];
+    /**
+     * Get an array of all available track IDs that are either specified by users or generated by the compiler.
+     * This can be used to call other API functions, e.g., `getTrack('track-1')`.
+     */
+    getTrackIds(): string[];
     getTracks(): TrackMouseEventData[];
     getTrack(trackId: string): TrackMouseEventData | undefined;
```

Comment:
For some reason there is more IDs in getTrackIds() than the number of tracks in getTracks(). I think for overlaid tracks the first track is not included in getTracks().